journal-of-the-marine-biological-association-of-the-united-kingdom.csl

```diff
@@ -249,6 +253,10 @@
           <text variable="publisher-place"/>
           <text variable="publisher"/>
         </group>
+        <group delimiter=" " prefix=", ">
```

Comment:
Make this prefix a delimiter on a group that includes the publisher and number-of-pages group